base/src/protocol_state.rs

```diff
@@ -23,6 +23,38 @@ pub struct ProtocolConstants {
     pub genesis_state_timestamp: BlockTime,
 }
 
+pub struct ProtocolConstantsTemp {
```

Comment:
It would be good to move this to consensus crate and name it something like `ConsensusConstants`: https://github.com/ChainSafe/mina-rs/pull/34/files#diff-1380b91b633a61c219d31250243405847967fde9e65ea2c4a4af6da87dc52753R13

Reply:
This temp struct was original from ProtocolConstans and is from [Constants](https://github.com/MinaProtocol/mina/blob/32f529d8d8d712a44ee75be66061ce08cbdc8924/docs/specs/consensus/README.md#3-constants). Should we move to consensus crate?

Reply:
Fixed.